### PR TITLE
Update executeScript snippets in migration guide

### LINKED ISSUE #1119: can't use alert() through scripting ReferenceError: alert is not defined
https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#executing-arbitrary-strings

the example on this page states that through use of the new .scripting API you can use alert in your extensions.
Is that true? I tried implementing it on my extension and it gave me a ReferenceError, the same error that usually comes up when you try to use the alert() function.

### PR DESCRIPTION
**Pull request by dotproto**
Updates the snippets in the [Executing arbitrary strings](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#executing-arbitrary-strings) section of the migration guide with snippets from the functional demo introduced in GoogleChrome/chrome-extensions-samples#626. 

Fixes #1119

[Preview page](https://deploy-preview-1129--developer-chrome-com.netlify.app/docs/extensions/mv3/intro/mv3-migration/#executing-arbitrary-strings)

### CODE REVIEW

File: site/en/docs/extensions/mv3/intro/mv3-migration/index.md [hunk 1]
```diff
@@ -355,106 +339,109 @@ chrome.tabs.executeScript({
...
+async function getCurrentTab() {/* ... */}
+let tab = await getCurrentTab();
+
 function showAlert() {
-  alert("test!");
+  alert('Function test alert');
 }
 
 chrome.scripting.executeScript({
+  target: {tabId: tab.id},
```
Comment (samthor) [inline site/en/docs/extensions/mv3/intro/mv3-migration/index.md:353]: This seems like the only practical change in this file (the rest is just reformatting).

File: site/en/docs/extensions/mv3/intro/mv3-migration/index.md [hunk 2]
```diff
@@ -279,48 +265,42 @@ With Manifest V3 the `executeScript()` method also moves to a different API.
...
+extension then uses this cached configuration to decide which features to enable.
 
+**Externalize logic with a remote service**—Consider migrating application logic from the extension
+to a remote web service that your extension can call. (Essentially a form of message passing.) This
+provides you the ability to keep code private and change the code on demand while avoiding the extra
+overhead of resubmitting to the Chrome Web Store.
 
 ### Executing arbitrary strings  {: #executing-arbitrary-strings }
 
 The `code` property from executeScript's
```
Comment (samthor) [inline site/en/docs/extensions/mv3/intro/mv3-migration/index.md:294]: I think you need to rewrite this line.

It should say something like "...previously, you used the `chrome.tabs` API to execute script in a parameter called `code`, but this has been replaced with calls in the `chrome.scripting` API which takes source files or function references.

File: site/en/docs/extensions/mv3/intro/mv3-migration/index.md [hunk 3]
```diff
@@ -355,106 +339,109 @@ chrome.tabs.executeScript({
...
+let tab = await getCurrentTab();
+
 function showAlert() {
-  alert("test!");
+  alert('Function test alert');
 }
 
 chrome.scripting.executeScript({
+  target: {tabId: tab.id},
   function: showAlert
```
Comment (samthor) [inline site/en/docs/extensions/mv3/intro/mv3-migration/index.md:354]: Same comment about this now being called `func:`.

**Review by awfuchs — APPROVED** [review]

File: site/en/docs/extensions/mv3/intro/mv3-migration/index.md (hunk 3 above)
Comment (dotproto) [reply site/en/docs/extensions/mv3/intro/mv3-migration/index.md:354]: Ah, I didn't realize this landed in stable. Thanks :)

File: site/en/docs/extensions/mv3/intro/mv3-migration/index.md (hunk 1 above)
Comment (dotproto) [reply site/en/docs/extensions/mv3/intro/mv3-migration/index.md:381]: Just force pushed to remove the line wrap commit. This should make the meaningful changes more obvious.

**Pull request merged**